### PR TITLE
remove pointer cursor from top-level menu (fixes #89)

### LINKED ISSUE #89: cursor should not be pointer when hovering over the Scene/Edit/Create
should be `default` since clicking Scene/Edit/Create doesn't do anything.

cursor should still be pointer when hovering over the inner menu items however

### PR DESCRIPTION
**Pull request by ngokevin**

### CODE REVIEW

File: src/css/dark.css [hunk 1]
```diff
@@ -110,6 +109,7 @@ input.number:focus {
 		}
 
 		#menubar .menu .options {
+      cursor: pointer;
```
Comment (ngokevin) [inline src/css/dark.css:120]: tabs?!

Comment (caseyyee) [inline src/css/dark.css:120]: Hehe, fixed the whitespace in https://github.com/aframevr/aframe-editor/pull/88 :P

Comment (ngokevin) [inline src/css/dark.css:120]: you can break that commit to another pr

Comment (caseyyee) [inline src/css/dark.css:120]: yeah, i'll just leave it for now since i've got a bunch of other changes in that PR related to changes in the CSS.

Comment (ngokevin) [inline src/css/dark.css:120]: should do your best to break it up. it's easy to pile in changes, but it's harder to review

Comment (caseyyee) [inline src/css/dark.css:120]: I'll remove that commit then.  everything else is related.

Comment (ngokevin) [inline src/css/dark.css:120]: although i'm a bit of a hypocrite with the monster PR haha https://github.com/aframevr/aframe-editor/pull/93 though i can't really break that up

Comment (ngokevin) [inline src/css/dark.css:120]: thanks, should free me to work on css too

Comment (caseyyee) [inline src/css/dark.css:120]: yeah, removed the commit, so we can work on the whitespace issue separately.
